Source/WebCore/Modules/highlight/Highlight.cpp

```diff
@@ -56,7 +56,7 @@ static void repaintRange(const SimpleRange& range)
 {
     // FIXME: Unclear precisely why we need to handle out of order cases here, but not unordered cases.
     auto sortedRange = range;
-    if (is_gt(treeOrder<ComposedTree>(range.start, range.end)))
+    if (WebCore::is_gt(treeOrder<ComposedTree>(range.start, range.end)))
```

Comment:
This code is in the `WebCore` namespace. Do we really need to add the `WebCore` prefix? Maybe you global replaced after trying `std`?

Reply:
It didn’t build without WebCore::. The compiler said it was ambiguous because of the same function coming from std. no idea why we have access to the std namespace without an explicit std::